### ISSUE TITLE
Plaid Integration refactor - intermittent UI not displayed

### ISSUE THREAD

**Issue by racitup** [frappe/erpnext#20242]
## Description of the issue
When navigating to the *Plaid Settings* page and enabling, sometimes the 4 configuration fields are not displayed. See screenshot.

## Context information (for bug reports)
```
ERPNext: v12.1.6 (version-12)
Frappe Framework: v12.0.16 (version-12)
```

## Steps to reproduce the issue
1. On a site without Plaid enabled, search for *Plaid Settings* and navigate to the page
2. Click the *Enabled* box

### Observed result
The 4 Plaid field options are not displayed:
![Screen Shot 2020-01-10 at 22 08 52](https://user-images.githubusercontent.com/18157195/72190449-7a4c9780-33f7-11ea-92e1-e23c97d429e8.png)

### Expected result
Sometimes the 4 fields are displayed. It seems to be related to how you navigate to the page. For example you can also get to the page by going to the desk, Integrations and Plaid Settings:
![Screen Shot 2020-01-10 at 22 22 17](https://user-images.githubusercontent.com/18157195/72190548-b3850780-33f7-11ea-9a10-3bd585698784.png)

## Additional information
Ubuntu 18.04, production install with install.py
Browser Chrome v79
Related to #18712

**Comment by Thunderbottom** [comment]
fixed with #20189

**Comment by Thunderbottom** [comment]
fixed with #20189
